fec/fec/static/js/calendar.js

```diff
@@ -339,6 +348,27 @@ Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
   }
 };
 
+Calendar.prototype.handleKeypress = function(e) {
+  if (e.keyCode === 13) {
+    $(e.target).click();
+  }
+};
+
+Calendar.prototype.controlPopoverFocus = function(e) {
```

Comment:
Hm yeah. What about `managePopoverControl`? It's basically implementing everything to make the popovers keyboard navigable.